docs/docs/tags.md

```diff
@@ -0,0 +1,3 @@
+# Tags index
```

Comment:
We've got a _Migration_ tag, that's uppercase _M_ and inconsistent with other tags, let's align it:

<img width="356" alt="image" src="https://user-images.githubusercontent.com/11164679/194704196-a33bed58-3416-4df3-85f6-1a0a658e09d8.png">